setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+packages = [
+    'robottelo',
+]
+
+requires = [
+    'ddt',
+    'fauxfactory',
+    'nailgun',
+    'paramiko',
+    'python-bugzilla',
+    'requests',
+    'selenium',
+]
+
+with open('README.rst', 'r') as f:
+    readme = f.read()
+
+setup(
+    name='robottelo',
+    version='0.1.0',
+    description='Robottelo is a test suite which exercises The Foreman.',
+    long_description=readme,
+    author=u'Satellite QE Team',
+    url='https://github.com/SatelliteQE/robottelo',
+    packages=packages,
```

Comment:
If I understand correctly (and I really should empirically check), `packages` lists out *every* single package that is supposed to be distributed. If so, the list here - a single package - is insufficient.